drivers/flash/flash_mcux_flexspi_hyperflash.c

```diff
@@ -418,6 +418,10 @@ static int flash_flexspi_hyperflash_write(const struct device *dev, off_t offset
 		return -EINVAL;
 	}
 
+#ifdef CONFIG_HAS_MCUX_CACHE
+	SCB_DisableICache();
+#endif
```

Comment:
There is a cache API in zephyr, shouldn't we use that? Or isn't that available for i.MX RT series?
```suggestion
	sys_cache_instr_disable();
```